_posts/03-02-01-Programming-Paradigms.md

```diff
@@ -3,51 +3,51 @@ isChild: true
 anchor:  programming_paradigms
 ---
 
-## Programming Paradigms {#programming_paradigms_title}
+## প্রোগ্রামিং এর ধরন {#programming_paradigms_title}
 
-PHP is a flexible, dynamic language that supports a variety of programming techniques. It has evolved dramatically over
-the years, notably adding a solid object-oriented model in PHP 5.0 (2004), anonymous functions and namespaces in
-PHP 5.3 (2009), and traits in PHP 5.4 (2012).
+পিএইচপি একটি পরিবর্তনশীল ও শক্তিশালী ল্যাঙ্গুয়েজ। এটি বিভিন্ন ধরনের প্রোগ্রামিং টেকনিক সমর্থন করে। গত কয়েক বছরে এর ইতিবাচক পরিবর্তন লক্ষ্য করা যায়,
+বিশেষ করে পিএইচপি ৫.০ (২০০৪) ভার্সন এ নির্ভরযোগ্য অবজেক্ট-ওরিয়েন্টেড মডেল এর সংযোজন, পিএইচপি ৫.৩ (২০০৯) ভার্সন এ অ্যাননিমাস ফাংশন এবং নেইমস্পেস এর সংযোজন,
```

Comment:
Let's keep the specific versions phrases in English. For example - 

- "পিএইচপি ৫.০" => "PHP 5.0"
- "পিএইচপি ৫.৩" =>"PHP 5.3"
- "পিএইচপি ৫.৪" => "PHP 5.4"

Reply:
Done